src/fheroes2/dialog/dialog_selectscenario.cpp

```diff
@@ -113,6 +117,35 @@ void ScenarioListBox::ActionListDoubleClick( Maps::FileInfo & )
     selectOk = true;
 }
 
+void ScenarioListBox::ActionListPressRight( Maps::FileInfo & info )
+{
+    // On some OS like Windows path contains '\' symbol. This symbol doesn't exist in the resources.
```

Comment:
I noticed a small typo in the comment and took the opportunity to improve the comment a little.

```suggestion
    // On some OSes like Windows, the path may contain '\' symbols. This symbol doesn't exist in the resources.
```

Reply:
Ah, thanks for spotting it!